go/vt/sqlparser/ast_format.go

```diff
@@ -116,17 +116,17 @@ func (node *Insert) Format(buf *TrackedBuffer) {
 		buf.astPrintf(node, "%s %v%sinto %v%v%v %v%v",
 			InsertStr,
 			node.Comments, node.Ignore.ToString(),
-			node.Table, node.Partitions, node.Columns, node.Rows, node.OnDup)
+			node.Table.Expr, node.Partitions, node.Columns, node.Rows, node.OnDup)
```

Comment:
hmm... don't we want to print the full AliasedTableExpr here?

Reply:
we remove the qualifier and alias it is not supported as syntax in insert statement for table

Reply:
TableName is SimpleTableExpr and here `node.Table.Expr` is also `SimpleTableExpr`